spyder/plugins/console/api.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Console Plugin API.
+"""
+
+# Local imports
+from spyder.plugins.console.widgets.main_widget import ConsoleWidgetActions

```

Comment:
Please don't forget to add here the other constants from `main_widget`.

Reply:
Fixed